dvc/command/experiments/show.py

```diff
@@ -473,7 +383,22 @@ def show_experiments(
         td.drop_duplicates("cols", ignore_empty=False)
 
     if pcp:
-        td.dropna("rows", how="all")
+        if drop is None:
+            drop = "Created"
+        else:
+            drop += "|Created"
+
+    if drop is not None:
+        td.drop(*[col for col in td.keys() if re.match(drop, col)])
```

Comment:
`subset` didn't work on the `drop` here? By the way, what is the priority relationship between these args? And I would prefer to give a Set to it as the order doesn't matter here.

Reply:
> subset didn't work on the drop here? 

The regex matching was moved outside per https://github.com/iterative/dvc/pull/7141#discussion_r777537175 . 

> By the way, what is the priority relationship between these args?

If combined `--only-changed` has the least priority, `--drop` comes next, and
`--keep` has the last word (from docs)

> And I would prefer to give a Set to it as the order doesn't matter here.
Done